### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by biportal** [peggyjs/peggy#519] [General]
Hi hildjj,

I’m new to sponsorships on GitHub and have a question: Would you consider applying for project sponsorship at https://github.com/sponsors? I’d like to support this specific project rather than individual contributors.

Thank you!

Best,
biportal

**Comment by hildjj** [comment]
I haven't put up sponsorship info on the Peggy project because I didn't write the code originally, and it didn't seem ethical for me to ask for money after we had to do the aggressive fork from the original code.  

However, if you'd like to sponsor my work on this library and others, you can go to https://github.com/sponsors/hildjj
